.github/ISSUE_TEMPLATE/project_thread.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Project Thread
```

Comment:
Is this supposed to replicate the Project Thread template? 🤔

Reply:
Yeah, we don't want to sync this. I guess we need to exclude this from syncing in https://github.com/WordPress/openverse